rules/integrations/o365/persistence_microsoft_365_global_administrator_role_assign.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2022/01/06"
+integration = "o365"
+maturity = "production"
+updated_date = "2022/01/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+In Azure Active Directory (Azure AD), permissions to manage resources are assigned using Roles. The Global Administrator
```

Comment:
Does this need to be updated to reflect 0365?

Reply:
Both use Azure AD to manage these Roles

Reply:
It is basically the same action, resulting in the same log, but coming from 2 different data sources